tests/PhpUnit/Oath/HotpTest.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace Tests\PhpUnit\Secrets;
```

Comment:
I love the tests! Thanks for adding them! Two small things I noticed about them:

1. The namespacing strategy used is not consistent with our other OC/SURFnet projects. We usually put the tests in the bundles, or alternatively put them in a /tests folder like you did, but still make them part of the bundle namespace.
2. The path structure should reflect the type of test. You chose PhpUnit, but this should be /acceptance/ or something like that.

Reply:
I didn't use the bundle namespace on purpose, because I wanted to decouple the end-to-end tests from the actual implementation.
I however moved the tests to an Acceptance folder because that's a good suggestion and changed the namespace accordingly.

Reply:
Alright, that makes perfect sense!